core/shared/src/main/scala/caseapp/core/help/HelpFormat.scala

```diff
@@ -17,12 +17,15 @@ import dataclass._
   sortedCommandGroups: Option[Seq[String]] = None,
   hidden: fansi.Attrs = fansi.Attrs.Empty,
   terminalWidthOpt: Option[Int] = None,
-  @since filterArgs: Option[Arg => Boolean] = None
+  @since filterArgs: Option[Arg => Boolean] = None,
+  @since filterArgsWhenShowHidden: Option[Arg => Boolean] = None,
+  @since hiddenGroupsWhenShowHidden: Option[Seq[String]] = None
```

Comment:
NIT Only a single `@since` for the newly added values is necessary, like
```suggestion
  @since filterArgsWhenShowHidden: Option[Arg => Boolean] = None,
  hiddenGroupsWhenShowHidden: Option[Seq[String]] = None
```

Reply:
(Having one for both will generate extra apply methods and constructors without just the last value, which are not really needed here.)

Reply:
Got it, fixed